DESCRIPTION

```diff
@@ -15,18 +15,21 @@ Imports:
     cli (>= 3.6.0),
     cowplot (>= 1.1.1),
     data.table (>= 1.14.8),
+    dplyr (>= 1.1.2),
```

Comment:
I'm probably going to write out this dependency when it goes into main, but don't have time at the moment and I'd rather let you get it out to the testers first